### PR TITLE
Fix KrigingAlgorithm docstring to use newer Optim API

### PR DESCRIPTION
**Pull request by dbarbier**
These changes had been forgotten in #17.

### CODE REVIEW

**Comment by jschueller** [comment]
question : how did this not make doctest fail before ?

**Comment by sofianehaddad** [comment]
@jschueller the method setOptimizer does not exist. So it is not parsed to doctoring (have a look at INSTALL/lib/python2.7/site-packages/openturns/metamodel.py ) and doctest pass

File: python/src/KrigingAlgorithm_doc.i.in [hunk 1]
```diff
@@ -157,7 +157,7 @@ Create the Kriging algorithm with the optimizer option:
 >>> bounds = ot.Interval(1e-2,1e2)
 >>> optimizer.setBoundConstraints(bounds)
```
Comment (sofianehaddad) [inline python/src/KrigingAlgorithm_doc.i.in]: The OptimizationSolver class does not provide such service
You should first define an optimization problem, fix bounds and set the problem to the solver

**Comment by dbarbier** [comment]
Branch is rebased and includes bugfixes from @sofianehaddad, thanks.

**Comment by sofianehaddad** [comment]
+1

**Comment by regislebrun** [comment]
+1

**Pull request merged**